spec/spec_test.go

```diff
@@ -1 +1,47 @@
 package spec
+
+import (
+  "encoding/json"
+  "testing"
+
+  assert "github.com/stretchr/testify/require"
+)
+
+func TestUnmarshal_Simple(t *testing.T) {
+  data := []byte(`{"type": "string"}`)
+  var schema Schema
+  err := json.Unmarshal(data, &schema)
+  assert.NoError(t, err)
+  assert.Equal(t, "string", schema.Type)
```

Comment:
And same in this file and `validate.go`. FYI, `gofmt -w <file>` if you didn't know about it already.